### DISCUSSION TITLE
Access parent's Fields from MorphManyRepater

### DISCUSSION THREAD

**Discussion by baloghdominik** [filamentphp/filament#2454] [Help] [answered]
Hi,

I can't figure out how to access the parent's field froma  MorphManyRepeater's record with Closure $get();

$get('name), $get('../name') and $get('../../name') does not work :/

Im trying to create a checkbox inside _**MorphManyRepeater::make('otherAddresses')**_ which if is checked, schould fill out _**MorphManyRepeater::make('otherAddresses')**_ fields from _**MorphManyRepeater::make('hqAddresses')**_ first and only record.

Currently I can't even access the parent's _**name field**_ from the _**MorphManyRepeater::make('otherAddresses')**_.

Any Idea?

```
` public static function form(Form $form): Form
    {
        return $form->schema([
            TextInput::make('name'),

            Repeater::make('otherAddresses')
                ->schema([
                    Checkbox::make('is_same_as_headquarters_address')
                        ->default(false)
                        ->reactive()
                        ->afterStateUpdated(
                            function ($state, \Closure $get, \Closure $set) {
                                if ($state) {
                                    $set('zip', $get('../../name'));
                                }
                            }
                        ),

                    TextInput::make('zip'),
                ]),
        ])->columns(1);
    }
`
```

**Comment by danharrin** [comment, marked as answer]
As mentioned on Discord, you need to be on the latest version for `../` to work.

**Reply by diazsasak** [reply]
Hi @danharrin , is there any way to do the otherwise? i have parent field to choose tax percentage and I want to update the items in repeater price based on the tax percentage. thank you
